packages/card-picker/package.json

```diff
@@ -42,12 +42,13 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
+    "ember-data": "^3.5.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^5.0.1",
-    "ember-source": "~3.4.0",
+    "ember-source": "~3.2.0",
```

Comment:
I'm assuming this is because of #450, but how is that related to this PR?

Reply:
Yes this was in relation to #450, more so the `ember-elsewhere` version bump than the `ember-let` issue. We can revert once we get a release of `@cardstack/edges`

Reply:
shouldn't that be unnecessary due to the use of `lerna`?

Reply:
eh... I don't think so 🤔 just because it explicitly depends on a particular version of edges in this package.json https://github.com/cardstack/cardstack/pull/451/files#diff-d6818a352bbaab3106246009a428b3e7R25

but then again I don't know if that is true 😂

Reply:
I tried to set `ember-source` back to `3.4.0` and do `lerna bootstrap` however the bottom edge simply stops working without throwing any errors.

Reply:
make sure to rebase. this should have been fixed by #453